theora uses inline assembly, can not build godot with msvc 2013 without theora=no

**Bugsquad note:** *There seems to be a confusion of which Visual Studio prompt to use when compiling the engine. The documentation needs to be more explicit about this subject.*

my env:

Intel Q6600 msvc 2013 ... windows 7 sp1 x64

As the title states, theora uses inline assembly, so scons + msvc 2013 x64 fails with: 

drivers\theora\cpu.c(59) : error C4235: nonstandard extension used : '_asm' keyword not supported on this architecture

I read on the net that msvc x64 doesn't support inline assembly... that would probably be the reason.

But, 32 bit also fails with theora in even though supposedly __asm is supported in it (not sure why, but it does)...

So, as far as i can tell, this isn't an issue with my compiler/computer, as I can build without theora x64 or x86... What i mean is, the issue is with every msvc (2013, and previous, and maybe 2015) compiler probably...

So, the compiling godot for windows tutorial on the wiki now (probably) won't work for anybody using msvc on x64... because of theora? Am i correct?

scons p=windows   //fails, above error (this should be x64 on my x64win, maybe i'm wrong?)
scons p=windows theora=no //successfull build
scons p=windows bits=32 //fails (not sure why, supposedly _asm is supported in 32bit but maybe not?)
scons p=windows bits=32 theora=no // build succeeds

--- a/platform/windows/detect.py
+++ b/platform/windows/detect.py
@@ -262,7 +262,7 @@ def configure(env):
 		env.Append(CCFLAGS=["/I"+DIRECTX_PATH+"/Include"])
 		env.Append(LIBPATH=[DIRECTX_PATH+"/Lib/x86"])
 		env['ENV'] = os.environ;
-		env["x86_opt_vc"]=env["bits"]!="64"
+		env["x86_opt_vc"]=True
 	else:
 
 		# Workaround for MinGW. See:

